recommendations-dashboard/terraform/modules/recommendations-dashboard/bigquery_schemas/recommendations_export_schema.sql

```diff
@@ -0,0 +1,34 @@
+        {"description":"Signed fractions of a second at nanosecond resolution of the span\\nof time. Durations less than one second are represented with a 0\\n`seconds` field and a positive or negative `nanos` field. For durations\\nof one second or more, a non-zero value for the `nanos` field must be\\nof the same sign as the `seconds` field. Must be from -999,999,999\\nto +999,999,999 inclusive.\\n","mode":"NULLABLE","name":"nanos","type":"INTEGER"}
+      ],"mode":"NULLABLE","name":"duration","type":"RECORD"}
+    ],"mode":"NULLABLE","name":"cost_projection","type":"RECORD"}
+  ],"mode":"NULLABLE","name":"primary_impact","type":"RECORD"},
+  {"description":"Output only. The state of the recommendation:\\n  STATE_UNSPECIFIED:\\n    Default state. Don't use directly.\\n  ACTIVE:\\n    Recommendation is active and can be applied. Recommendations content can\\n    be updated by Google.\\n    ACTIVE recommendations can be marked as CLAIMED, SUCCEEDED, or FAILED.\\n  CLAIMED:\\n    Recommendation is in claimed state. Recommendations content is\\n    immutable and cannot be updated by Google.\\n    CLAIMED recommendations can be marked as CLAIMED, SUCCEEDED, or FAILED.\\n  SUCCEEDED:\\n    Recommendation is in succeeded state. Recommendations content is\\n    immutable and cannot be updated by Google.\\n    SUCCEEDED recommendations can be marked as SUCCEEDED, or FAILED.\\n  FAILED:\\n    Recommendation is in failed state. Recommendations content is immutable\\n    and cannot be updated by Google.\\n    FAILED recommendations can be marked as SUCCEEDED, or FAILED.\\n  DISMISSED:\\n    Recommendation is in dismissed state.\\n    DISMISSED recommendations can be marked as ACTIVE.\\n","mode":"NULLABLE","name":"state","type":"STRING"},
+  {"description":"Ancestry for the recommendation entity\\n","fields":[
+    {"description":"Organization to which the recommendation project\\n","mode":"NULLABLE","name":"organization_id","type":"STRING"},
+    {"description":"Up to 5 levels of parent folders for the recommendation project\\n","mode":"REPEATED","name":"folder_ids","type":"STRING"}
+  ],"mode":"NULLABLE","name":"ancestors","type":"RECORD"},
+  {"description":"Insights associated with this recommendation. A project insight is represented as\\nprojects/[PROJECT_NUMBER]/locations/[LOCATION]/insightTypes/[INSIGHT_TYPE_ID]/insights/[insight_id]\\n","mode":"REPEATED","name":"associated_insights","type":"STRING"},
+  {"description":"Additional details about the recommendation in JSON format\\n","mode":"NULLABLE","name":"recommendation_details","type":"STRING"},
+  {"description":"Priority of the recommendation:\\n  PRIORITY_UNSPECIFIED:\\n    Default unspecified priority. Don't use directly.\\n  P4:\\n    Lowest priority.\\n  P3:\\n    Second lowest priority.\\n  P2:\\n    Second highest priority.\\n  P1:\\n    Highest priority.\\n","mode":"NULLABLE","name":"priority","type":"STRING"}
+]
+
```

Comment:
yeah, the formatting for the `fields` here is just tough... I'm not sure what to do with it